libvcx/Cargo.toml

```diff
@@ -23,6 +23,7 @@ fatal_warnings = []
 
 [dependencies]
 once_cell = { version = "1.15" }
+cfg-if = { version = "1.0" }
```

Comment:
A nitpick, but personally, I would prefer staying consistent with the versioning syntax (or making it consistent)